tests/assembly/test.ts

```diff
@@ -19,7 +19,7 @@ export function runTest(): void {
     original.uint8array = base64.decode("aGVsbG8sIHdvcmxkIQ==");
     original.u128Val = new u128(128);
     original.arr = [["Hello"], ["World"]];
-    original.uint8arrays = Array.create<Uint8Array>(2);
+    original.uint8arrays = new Array<Uint8Array>(2);
```

Comment:
is Array.create still available? (in case some clients have this code)

Reply:
It's still there but deprecated and you'll get a warning when compiling.